### DISCUSSION TITLE
[BUG] UFW 开启后问题

### DISCUSSION THREAD

**Discussion by ting7777** [1Panel-dev/1Panel#6718] [Q&A]
### 1Panel 版本

v1.10.18-lts

### 问题描述

1. 这里就很一直在加载中。
<img width="532" alt="image" src="https://github.com/user-attachments/assets/7fa8c763-4ec4-459c-b429-74b79885a7c0">

2. 开机会很久才进的到1Panel
 - systemd-analyze blame
    > 5min 202ms networking.service

 - systemd-analyze critical-chain
   > graphical.target @5min 3.686s
      -multi-user.target @5min 3.685s
        -docker.service @5min 757ms +2.928s
          -containerd.service @5min 373ms +376ms
            -network.target @5min 358ms
              -systemd-resolved.service @345ms +116ms
                -systemd-networkd.service @269ms +72ms
                  -network-pre.target @248ms
                    -nftables.service @179ms +68ms
                      -systemd-journald.socket @169ms
                        -system.slice @158ms
                          --.slice @158ms

### 重现步骤

开启 UFW 后，只要 "ufw disable", 重启就没有问题了

### 期待的正确结果

只要开启UFW 就会有这样的问题，请问是设定问题吗？

**Comment by wanghe-fit2cloud** [comment]
可以先通过 `journalctl -u 1panel.service -exf` 命令查看详细启动日志，确认网络启动阶段是否有延迟。
